src/plots/smith/constants.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+module.exports = {
+    attr: 'subplot',
+    name: 'smith',
+
+    axisNames: ['imaginaryaxis', 'realaxis'],
+    axisName2dataArray: {imaginaryaxis: 'theta', realaxis: 'r'},
+
+    layerNames: [
+        'draglayer',
+        'plotbg',
+        'backplot',
+        'angular-grid',
+        'radial-grid',
+        'frontplot',
+        'angular-line',
+        'radial-line',
+        'angular-axis',
+        'radial-axis'
+    ],
+
```

Comment:
Please remove those that are not needed.

Reply:
2ca14f9da61b05283dd62ebc42564ea3661769e1